wellregistry/registry/admin.py

```diff
@@ -48,7 +44,8 @@ class RegistryAdmin(admin.ModelAdmin):
     @staticmethod
     def site_id(obj):
         """Constructs a site id from agency code and site number."""
-        return f"{obj.agency_cd}:{obj.site_no}"
+        # The obj field agency_cd is the AgencyLovLookup model, retrieve agency_cd from the model
+        return f"{obj.agency_cd.agency_cd}:{obj.site_no}"
```

Comment:
"agency_cd.agency_cd" did the primary key have to change for agency with the lookup table?

Reply:
The primary key of agency_lov table did not change, it is still agency_cd. The registry model was updated to have
the agency model as the field instead of a agency_cd string. Hence accessing the field via agency_cd.agency_cd